### PR TITLE
adds z-index tokens

### PR DESCRIPTION
**Pull request by brianrclay**
Adding z-index tokens. I tried to build the tokens but it's failing and I'm not sure how to fix it. The tokens are based on Bootstrap's model. Totally open to change it up if you guys feel something is better. This is just a first pass.

@phantomxc @jgerlach

### CODE REVIEW

File: src/tokens/zIndex.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  DropdownMenu: core.ZIndex.Overlay1,
+  // Message Box
+  MessageBox: core.ZIndex.Overlay3,
+  // Modal
+  Modal: core.ZIndex.Overlay5,
+  // Tooltip
+  Tooltip: core.ZIndex.Overlay7,
+})
+
+const dark = (core) => ({})
```
Comment (jgerlach) [inline src/tokens/zIndex.js]: I'm not sure what error you are having when building, but the dark tokens are going to be empty unless you spread the light tokens in like we do with the Spacing tokens like this:
```
const dark = (core) => ({
  ...light(core),
})
```

Comment (jgerlach) [reply src/tokens/zIndex.js]: also, make sure to run `npm build:docs` and `npm run json`

Comment (brianrclay) [reply src/tokens/zIndex.js]: Fixed that @jgerlach

Comment (jgerlach) [reply src/tokens/zIndex.js]: Looks good to me. I'm going to approve it. You might want Cameron to take a look before you publish it.

**Comment by run-time** [comment]
this MR was too outdated so I'm closing it (these tokens will be added to the 10.0 release)

**Pull request closed without merging**